performance/src/include/firebase/performance.h

```diff
@@ -0,0 +1,60 @@
+#ifndef FIREBASE_PERFORMANCE_CLIENT_CPP_SRC_INCLUDE_FIREBASE_PERFORMANCE_H_
+#define FIREBASE_PERFORMANCE_CLIENT_CPP_SRC_INCLUDE_FIREBASE_PERFORMANCE_H_
+
+#include "firebase/app.h"
+#include "firebase/performance/http_metric.h"
+#include "firebase/performance/trace.h"
+
+/// @brief Namespace that encompasses all Firebase APIs.
+namespace firebase {
+
+/// @brief Firebase Performance API.
+///
+/// See <a href="/docs/perf-mon">the developer guides</a> for general
```

Comment:
I think here might be a good place to put a notice about this being experimental?

Reply:
Done